src/Data/List/Relation/Unary/All.agda

```diff
@@ -116,17 +116,17 @@ unzip : All (P ∩ Q) ⊆ All P ∩ All Q
 unzip = unzipWith id
 
 module _(S : Setoid a ℓ) {P : Pred (Setoid.Carrier S) p} where
-  open Setoid S renaming (Carrier to C; refl to refl₁)
+  open Setoid S renaming (refl to ≈-refl)
   open SetoidMembership S
 
-  tabulateₛ : (∀ {x} → x ∈ xs → P x) → All P xs
-  tabulateₛ {[]}     hyp = []
-  tabulateₛ {x ∷ xs} hyp = hyp (here refl₁) ∷ tabulateₛ (hyp ∘ there)
+  tabulateₛ : ∀∈ P xs → All P xs
+  tabulateₛ {[]}    hyp = []
+  tabulateₛ {_ ∷ _} hyp = hyp (here ≈-refl) ∷ tabulateₛ (hyp ∘ there)
 
-tabulate : (∀ {x} → x ∈ₚ xs → P x) → All P xs
+tabulate : ∀∈ₚ P xs → All P xs
 tabulate = tabulateₛ (≡.setoid _)
 
-self : ∀ {xs : List A} → All (const A) xs
+self : ∀ {xs} → All (const A) xs
```

Comment:
Do we need `{xs}`?

Reply:
Er.... it turns out not to be possible to remove it, because we then lose the correlation between the type `List A` of `xs` and the `A` mentioned in the type, and restoring that telescopic relationship requires more syntax than we save! Unless you can find a good solution?